telemetry/specs/INT.mdk

```diff
@@ -955,7 +945,7 @@ INT Shim Header for TCP/UDP, INT type is hop-by-hop:
     0                   1                   2                   3
     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-   |     Type=1    |   Reserved    |  Length = 8   |   Reserved    |
+   |     Type=1    |   Reserved    |  Length = 10  |   Reserved    |
```

Comment:
length = 8 looks to be right. why is it 10?

Reply:
Thanks for catching this. I started adding switch3 metadata, hence bumped up length to 10. But then realized that this switch3 is INT sink in this example, forgot to revert length back to 8.

Will change this back to 8.